doc/classes/Array.xml

```diff
@@ -536,6 +536,7 @@
 				Removes an element from the array by index. If the index does not exist in the array, nothing happens. To remove an element by searching for its value, use [method erase] instead.
 				[b]Note:[/b] This method acts in-place and doesn't return a value.
 				[b]Note:[/b] On large arrays, this method will be slower if the removed element is close to the beginning of the array (index 0). This is because all elements placed after the removed element have to be reindexed.
+				[b]Note:[/b] [param position] cannot be negative. To remove the last element from the array, use [code]arr.remove_at(arr.size()-1)[/code].
```

Comment:
```suggestion
				[b]Note:[/b] [param position] cannot be negative. To remove an element relative to the end of the array, use [code]arr.remove_at(arr.size()-(i + 1))[/code]. To remove the last element from the array, use [code]arr.resize(arr.size()-1)[/code].
```

Reply:
Thanks. I manually applied this suggestion so I could also fix the typo: `returning the *vale*`.